### ISSUE TITLE
Implement DataController.insertContent()

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5#3861]
Needed for ckeditor/ckeditor5-clipboard#1.

It seems that it must accept _view_ document fragment, because it will need to convert it to the model using the context of insertion. And this becomes part of the "insert content" logic, so should most likely be done inside this method.

BTW, it made me think that this method would make sense on the `DataController`. And I kinda like this idea. We've been thinking where to move the composer and I can see that `modifySelection()` makes for the `EditingController` and `deleteContent()` together with `insertContent()` could perhaps end in the the `DataController`. Note also that we'll need something like `getSelectedContent()` for the copy and cut operations (so `deleteContent()` would become `deleteSelectedContent()`).

OTOH, I know what you're going to say – those methods look weird on the `DataController`. I agree... I'd have to get used to them. But consider this – let's say we'd like to move `insertContent()` to the `Document`. But `insertContent()` requires (as explained above) a knowledge about `DataController`. So how could it be implemented on the `Document`?

tl;dr:
- `DataController#insertContent()` – paste and I'm sure it'll be used in many ways in many cases by developers,
- `DataController#deleteSelectedContent()` – delete feature,
- `DataController#getSelectedContent()` – copy, cut and this has been also requested many times in the past for various integration scenariosm
- `EditingController#modifySelection()`

Does it make any sense or no sense at all?
